manifest_spec.yaml.tmpl

```diff
@@ -0,0 +1,17 @@
+image: REGISTRY/sonobuoy:TAG
+manifests:
+  -
+    image: REGISTRY/sonobuoy-amd64:TAG
+    platform:
+      architecture: amd64
+      os: linux
+  -
+    image: REGISTRY/sonobuoy-arm64:TAG
+    platform:
+      architecture: arm64
+      os: linux
+  -
+    image: REGISTRY/sonobuoy-win-amd64:TAG
```

Comment:
Should have removed this section.

Moving to this manifest version instead of the `--from-args` for manifests enables easy addition of Windows in the future though.

Reply:
Actually added a WIN_ONLY prefix so we can optionally remove the windows stuff or add it, depending on the options given to MAKE.